contrib/mesos/docs/issues.md

```diff
@@ -198,6 +198,16 @@ The scheduler also offers `/debug` API endpoints that may be useful:
 All of the issues in the above section also apply to the Kubernetes-Mesos DCOS package builds.
 The issues listed in this section apply specifically to the Kubernetes-Mesos DCOS package available from https://github.com/mesosphere/multiverse.
 
+### Etcd
+
+The default configuration of the DCOS Kubernetes package launches an internal etcd process **which only persists the cluster state in the task's sandbox**. While this is simpler for the first steps with Kubernetes-Mesos, it means that any cluster state is lost when the Kubernetes-Mesos Docker container is restarted.
```

Comment:
I suggested using a word other than task because I think it's not very meaningful unless you've got a ton of context like we do.

> Installing the Kubernetes DCOS service using the default configuration options results in the creation of an internal etcd process that stores the cluster state inside the same service container as the rest of the Kubernetes "master" service components. While this simplifies the process of getting started with Kubernetes on DCOS it also means that the Kubernetes cluster state is completely lost if/when the Kubernetes service container fails or is otherwise terminated. As a consequence of such an event DCOS will attempt to restart the Kubernetes service whereby a new service container, including a new internal etcd process, will be created as a result.